shared/src/main/java/com/faforever/neroxis/util/ImageUtil.java

```diff
@@ -288,25 +289,6 @@ private static byte[] getCompressedDDSImageBytes(int size, ByteBuffer imageByteB
         return allBytes;
     }
 
-    private static byte[] getRawDDSImageBytes(int size, ByteBuffer imageByteBuffer) {
-        DDSHeader ddsHeader = new DDSHeader();
-        ddsHeader.setWidth(size);
-        ddsHeader.setHeight(size);
-        ddsHeader.setRGBBitCount(32);
-        ddsHeader.setRBitMask(0x000000FF);
-        ddsHeader.setGBitMask(0x0000FF00);
-        ddsHeader.setBBitMask(0x00FF0000);
-        ddsHeader.setABitMask(0xFF000000);
-        ddsHeader.toBytes();
-        byte[] headerBytes = ddsHeader.toBytes();
-        byte[] imageBytes = imageByteBuffer.array();
-        int headerLength = headerBytes.length;
-        int imageLength = size * size * 4;
-        byte[] allBytes = Arrays.copyOf(headerBytes, headerLength + imageLength);
-        System.arraycopy(imageBytes, 0, allBytes, headerLength, imageLength);
-        return allBytes;
-    }
```

Comment:
Why was this method deleted?

Reply:
It was unused.

Reply:
I noticed that this also uses ABGR order. So instead of thinking a lot about if I need to change the byte order there as well I deleted it. Without the method being used I can't really decide which order is appropriate and I didn't want anyone to use it in the future, expecting it to behave similar to `writeRawDDS`